packages/aws-cdk-lib/aws-elasticloadbalancingv2/test/alb/listener.test.ts

```diff
@@ -1933,6 +1934,139 @@ describe('tests', () => {
     });
 
   });
+
+  describe('Mutual Authentication', () => {
+    test('Mutual Authentication settings with all propeties', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'Stack');
+      const lb = new elbv2.ApplicationLoadBalancer(stack, 'LB', { vpc });
+      const bucket = new s3.Bucket(stack, 'Bucket');
+
+      const trustStore = new elbv2.TrustStore(stack, 'TrustStore', {
+        bucket,
+        key: 'dummy.pem',
+      });
+
+      // WHEN
+      lb.addListener('Listener', {
+        protocol: elbv2.ApplicationProtocol.HTTPS,
+        certificates: [importedCertificate(stack)],
+        mutualAuthentication: {
+          ignoreClientCertificateExpiry: true,
+          mutualAuthenticationMode: elbv2.MutualAuthenticationMode.VERIFY,
```

Comment:
Should we add in checks to verify that `PASSTHROUGH` and `OFF` authentication modes also synthesize correctly?

Reply:
I've added unit tests.